runtime/node/src/customaction/customActionComponentRegistration.ts

```diff
@@ -0,0 +1,14 @@
+import { ComponentRegistration } from 'botbuilder';
+import { ComponentDeclarativeTypes, DeclarativeType, ResourceExplorer } from 'botbuilder-dialogs-declarative';
+import { MultiplyDialog } from './multiplyDialog';
+
+export class CustomActionComponentRegistration extends ComponentRegistration implements ComponentDeclarativeTypes {
```

Comment:
add some inline comments to explain what thi is doing and how a developer would customize it to create their own